packages/core/migrations/2_deploy_finder.js

```diff
@@ -5,4 +5,5 @@ module.exports = async function (deployer, network, accounts) {
   const keys = getKeysForNetwork(network, accounts);
 
   await deploy(deployer, network, Finder, { from: keys.deployer });
+  // test comment
```

Comment:
should this still be here? I'm guessing you added it to validate that CI steps ran correctly.